### PR TITLE
Arch installation procedure update

### PR DESCRIPTION
**Pull request by aarohijohal**
Recent repo changes + command line format requirement needs these changes to make the installation instructions to work on Arch Linux.

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -55,9 +55,11 @@ Under **Arch** Linux :
 
 ```sh
 pacman -S nodejs npm qpdf imagemagick pdfminer tesseract python-pip
-pip install camelot-py[cv] pdfminer.six numpy pillow scikit-image PyPDF2
```
Comment (marianorodriguez) [inline docs/installation.md:58]: why removing pdfminer.six ?

Comment (aarohijohal) [reply docs/installation.md:58]: The line above: 
`pacman -S nodejs npm qpdf imagemagick pdfminer tesseract python-pip`
already installs pdfminer.
The above package is based on pdfminer.six [link](https://www.archlinux.org/packages/community/any/python-pdfminer/).
However, there's a typo: let me fix.

Comment (aarohijohal) [reply docs/installation.md:58]: > However, there's a typo: let me fix.
Done in fead78d

Comment (marianorodriguez) [reply docs/installation.md:58]: nice! I didn't realize it was on pacman

**Pull request merged**